### PR TITLE
Make schemas

### PR DESCRIPTION
**Pull request by henrytso**

### CODE REVIEW

File: app.py
Comment (kq-li) [inline app.py]: these imports are extraneous, i'll have to remove them from uplift too

File: schema.py
Comment (kq-li) [inline schema.py]: prefer `about_short`

File: schema.py
Comment (kq-li) [inline schema.py]: prefer `is_open`

File: schema.py
Comment (kq-li) [inline schema.py]: might want more descriptive name for the actual query argument -- `pay_method`?

File: schema.py
Comment (kq-li) [inline schema.py]: move `OperatingHoursType,` to its own line

File: scraper.py
Comment (kq-li) [inline scraper.py]: this is slightly misnamed -- scraping implies extracting data from HTML, you can just do this directly in `data.py` probably with the python `json` module

**Review by kq-li — CHANGES_REQUESTED** [review]
you're gonna need resolvers (one of the few things that are actually documented) to get any data from your queries -- probably should do that before any merges

**Comment by young-k** [comment]
In the future, make sure to rebase commits like "Make style changes", here it's fine since it's the two of you guys working together.

File: scraper.py
Comment (young-k) [reply scraper.py]: +1 here, we don't need a `scraper.py`, it can be done in `data.py`

**Comment by young-k** [comment]
@henrytso @cswenberg can you delete `scraper.py`, i'll approve after

**Comment by ilikesymmetry** [comment]
Made resolvers @kq-li

File: app.py
Comment (kq-li) [inline app.py]: indent an extra 2 spaces when inside parens/brackets

File: data.py
Comment (kq-li) [inline data.py]: extra 2 spaces here too

File: data.py
Comment (kq-li) [inline data.py]: and here

File: data.py
Comment (kq-li) [inline data.py]: paren to its own line

File: data.py
Comment (kq-li) [inline data.py]: uhh this should probably be an exception

File: data.py
Comment (kq-li) [inline data.py]: -2 spaces here

File: data.py
Comment (kq-li) [inline data.py]: +2 spaces here

File: data.py
Comment (kq-li) [inline data.py]: can just set the variable equal to the value `any(...)` since that's already the boolean you want

File: data.py
Comment (kq-li) [inline data.py]: indentation

File: data.py
Comment (kq-li) [inline data.py]: indentation

File: data.py
Comment (kq-li) [inline data.py]: global declaration should prob be at top of function body

File: data.py
Comment (kq-li) [inline data.py]: indentation

File: data.py
Comment (kq-li) [inline data.py]: global to top

File: schema.py
Comment (kq-li) [inline schema.py]: 2 space indentation in this file

File: schema.py
Comment (kq-li) [inline schema.py]: luckily the graphene `ObjectType` constructor automatically does exactly what all these constructors do so you can leave them out

File: schema.py
Comment (kq-li) [inline schema.py:44]: 1 line between classes is fine

File: schema.py
Comment (kq-li) [inline schema.py]: what does this comment mean?

File: schema.py
Comment (kq-li) [inline schema.py]: should these fields also be required?

File: schema.py
Comment (kq-li) [inline schema.py]: `EateryType` to new line

File: schema.py
Comment (kq-li) [inline schema.py]: remove comment + extra whitespace

File: schema.py
Comment (kq-li) [inline schema.py]: i believe you'll also need a resolver for operating hours

**Review by kq-li — COMMENTED** [review]
looks well fleshed out, just some style things + one resolver seems to be missing but overall good work! make sure you've tested the functionality, the default endpoint for the webserver has a nice "graphiql" interface that lets you query easily

File: data.py
Comment (young-k) [reply data.py]: HTTP PARSE IMAGE URL FAILED LOOOOL

File: schema.py
Comment (young-k) [reply schema.py]: "EVENT"

**Comment by young-k** [comment]
rebase some of the commits away, like the two "Update README" commits

File: data.py
Comment (kq-li) [inline data.py]: `Data` should be a static singleton class and not actually instantiated, otherwise you'll get a new object every update

File: data.py
Comment (kq-li) [inline data.py]: we python 3 bois out here, `print(...)` plz

File: data.py
Comment (kq-li) [inline data.py]: remove

File: app.py
Comment (kq-li) [reply app.py]: here too

File: app.py
Comment (kq-li) [inline app.py]: here should be another extra so 8 spaces

File: data.py
Comment (kq-li) [inline data.py:144]: remove extra line

File: schema.py
Comment (kq-li) [inline schema.py]: remove extra line

File: schema.py
Comment (kq-li) [inline schema.py]: these constructors are unnecessary -- graphene provides a default constructor that does exactly this (for all the `...Type`s)

File: schema.py
Comment (kq-li) [inline schema.py]: consider removing this, functionality seems redundant -- make sure you consult with ios/android devs and find out exactly what they need to be able to query for with what inputs

**Comment by ilikesymmetry** [comment]
@kq-li pls I need this

**Review by young-k — APPROVED** [review]
lgtm

**Pull request merged**
